client/src/components/App.jsx

```diff
@@ -56,16 +56,19 @@ class App extends React.Component {
     let products;
 
     axios.get(`/api/products/${productId}`)
-      .then((res) => {
-        products = res.data;
-      })
-      .then(() => axios.get(`/api/wishlist/${productId}`))
       .then((res) => {
         this.setState({
-          products,
-          wishList: res.data.products,
-        });
+          products: res.data.rows
```

Comment:
I think best practice would be sending the .rows data in your response, vs. filtering it out in the client, especially if your client can't use/doesn't need to see the other data.